projects/js-packages/components/components/product-detail/style.module.scss

```diff
@@ -0,0 +1,96 @@
+.wrapper {
+	height: 100%;
+	padding: calc( var( --spacing-base ) * 8 );
+	position: relative;
+	min-width: calc( var( --spacing-base ) * 48 );
+
+	&.is-card {
+		background-color: var( --jp-white );
+		border: 1px solid var( --jp-gray );
+		border-radius: var( --jp-border-radius );
+		box-shadow: 0 2px 6px var( --product-card-shadow ), 0 1px 2px var( --product-card-shadow );
+
+		.card-header {
+			border-radius: var( --jp-border-radius ) var( --jp-border-radius ) 0 0;
+		}
+	}
+}
+
+.card-header {
+	display: flex;
+	align-items: center;
+	background: var( --jp-black );
+	color: var( --jp-white );
+	padding: 0 var( --spacing-base );
+	height: calc( var( --spacing-base ) * 4 );
```

Comment:
I think that we should avoid relating `spacing` with `size.`

If we need to force some value, let's put it directly; if not, we define the padding, and the height will be `auto.`

Spacing and size are already different topics at Material UI (an example):

https://mui.com/system/spacing/#notation
https://mui.com/system/sizing/

Reply:
> If we need to force some value, let's put it directly; if not, we define the padding, and the height will be auto.

Sounds good to me.

> Spacing and size are already different topics at Material UI (an example):

That's nice. maybe something to implement for RNA too

Reply:
addressed here https://github.com/Automattic/jetpack/pull/23842/commits/d7a93379d6f7e876d9fef1df8bc89fec02af1ad9